src/components/Form/DynamicForm.tsx

```diff
@@ -55,6 +62,8 @@ const DynamicForm = ({ currentForm, machineValues, control, errors }) => {
                                         defaultValue={defaultValue}
                                         data-cy={`testClaimReview${fieldName}`}
                                         extraProps={extraProps}
+                                        editorRef={editorRef}
```

Comment:
suggestion: instead of using dependency injection, create a provider for the editor